src/access-control/values.yaml

```diff
@@ -4,39 +4,11 @@
 global:
   loadbalancerURL: ""
   database:
-    postgres:
-      ## - installed = true if installed within cluster
-      installed: true
-      ## - protocol to use for connection
-      protocol: postgres
-      ## - host array for external
-      hosts:
-        - postgres:5432
-      ## - secret name containing external values
-      secretName: ""
-      ## - key within secret containing username
-      userKey: ""
-      ## - key within secret containing password
-      passwordKey: ""
-      ## - extra arguments set to connection string
-      extraArgs: ""
     mongo:
-      installed: false
-      protocol: mongodb
-      # --  provide default values if mongo.installed is set to false
-      hosts:
-      - h1
-      - h2
-      secretName: "xx"
-      userKey: "xx"
-      passwordKey: "xxx"
-      extraArgs: "xx"
-    timescaledb:
       installed: true
-      protocol: "jdbc:postgresql"
+      protocol: mongodb
```

Comment:
why is protocol changed ?

Reply:
it has not changed, i removed global.database.timescaledb and global.database.postgres field from access-control as it does not require them.
Github probably did not show the difference correctly